GameProject/Assets/OpenWorld Assets/Scripts/ThirdPersonCharacterController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ThirdPersonCharacterController : MonoBehaviour
+{
+
+    [SerializeField] private float _moveSpeed;
+    [SerializeField] private float _rotateSpeed;
+    //private float rotX;
+    //private float rotY;
+    [SerializeField] private float _rotZ;
+    private Camera _cam;
```

Comment:
Abbreviations are generally frowned upon. Consider using proper names (same goes for `_rotZ` and other such variables).